### PR TITLE
Get rid of integrity validations

### LINKED ISSUE #1239: `Unsafe` suffix for functions has different meanings
We have several `*Unsafe` functions defined in our project

```
$ ag 'Unsafe ::'
packages/xod-project/src/patch.js
248:  'getNodeByIdUnsafe :: NodeId -> Patch -> Node',
499:  'getLinkByIdUnsafe :: LinkId -> Patch -> Link',
750:  'getCommentByIdUnsafe :: CommentId -> Patch -> Comment',

packages/xod-project/src/xodball.js
35:  'fromXodballDataUnsafe :: Object -> Project',

packages/xod-project/src/project.js
318:  'getPatchByPathUnsafe :: PatchPath -> Project -> Patch',
700:  'assocPatchUnsafe :: PatchPath -> Patch -> Project -> Project',
734:  'assocPatchListUnsafe :: [Patch] -> Project -> Project',
948:  'getPatchByNodeIdUnsafe :: NodeId -> Patch -> Project -> Patch',
1105:  'changeNodeTypeUnsafe :: PatchPath -> NodeId -> PatchPath -> Project -> Project',

packages/xod-project/src/utils.js
214:  'getTypeFromLiteralUnsafe :: DataValue -> DataType',

packages/xod-arduino/src/transpiler.js
189:  'getNodePinsUnsafe :: Node -> Project -> [Pin]', — throws
```

For most of them it means "try doing stuff and throw an exception if it's invalid".
But for others, like `assocPatchUnsafe`, it means "just do that thing and don't care if the result is not 100% valid"(resulting project could have some dead references or links with incompatible types, but our code won't crash)

- Leave `Unsafe` suffix only for functions that will throw exceptions
- For others, invent another suffix(?) and rename them
   - `assocPatchUnsafe :: PatchPath -> Patch -> Project -> Project`
   - `assocPatchListUnsafe :: [Patch] -> Project -> Project`
   - `changeNodeTypeUnsafe :: PatchPath -> NodeId -> PatchPath -> Project -> Project`

### LINKED ISSUE #1369: Get rid of redundant integrity validations
### Rationale

We have some inconsistencies on who validates the program integrity and when. It causes redundant validity checks in (a) actions → (b) assoc/dissoc in xod-project → (c) re-validation on program change. We can get rid of this redundancy by removing the integrity checks from (a) and (b), leaving only (c).

### Acceptance criteria

- [x] Do not check for integrity breaks in actions
- [x] Do not check for integrity breaks in assoc/dissoc nodes and links in `xod-project`

### PR DESCRIPTION
**Pull request by brusherru**
It closes #1369 and closes #1239

### CODE REVIEW

File: packages/xod-client/src/editor/actions.js [hunk 1]
```diff
@@ -501,6 +501,7 @@ export const pasteEntities = event => (dispatch, getState) => {
   );
 
   const entitiesToPaste = R.compose(
+    R.tap(a => console.log('wtf?', a)),
```
Comment (evgenykochetkov) [inline packages/xod-client/src/editor/actions.js]: 😁

Comment (brusherru) [reply packages/xod-client/src/editor/actions.js]: Oops 😱

**Comment by evgenykochetkov** [comment]
Does this also take of https://github.com/xodio/xod/issues/1239?

File: packages/xod-client/src/project/reducer.js [hunk 2]
```diff
@@ -441,8 +440,7 @@ export default (state = {}, action) => {
         R.pipe(
           XP.omitLinks(conflictingLinks),
           XP.assocNode(busNode),
-          XP.assocLink(link),
-          explodeEither
```
Comment (evgenykochetkov) [inline packages/xod-client/src/project/reducer.js]: So many pointless `explode`s are gone 👍🏻

**Comment by nkrkv** [comment]
May we run `refmt` over `master` and make it a separate PR *before* merging this PR? So that we get rid of false diffs.

**Pull request merged**